src/script/hydra-create-user

```diff
@@ -25,27 +25,37 @@ exists, roles are added to the existing roles unless --wipe-roles is
 specified.  If --rename-from is given, the specified account is
 renamed.
 
-* PASSWORD HASH
-The password hash should be an Argon2id hash, which can be generated
-via:
+* Specifying Passwords
+
+** Specifying a Hash
+
+You can generate a password hash and provide the hash as well. This
+is useful so a user can send the administrator their password pre-hashed,
+allowing the user to get their preferred password without exposing it
+to the administrator.
+
+Hydra uses Argon2id hashes, which can be generated like so:
 
     $ nix-shell -p libargon2
-    [nix-shell]$ argon2 "$(LC_ALL=C tr -dc '[:alnum:]' < /dev/urandom | head -c16)" -id -t 3 -k 262144 -p 1 -l 16 -e
+    [nix-shell]$ tr -d \\\\n | argon2 "$(LC_ALL=C tr -dc '[:alnum:]' < /dev/urandom | head -c16)" -id -t 3 -k 262144 -p 1 -l 16 -e
```

Comment:
I can confirm these updated instructions work for me.

I did:

```
$ tr -d \\\n | argon2 "$(LC_ALL=C tr -dc '[:alnum:]' < /dev/urandom | head -c16)" -id -t 3 -k 262144 -p 1 -l 16 -e
hii
$argon2id$v=19$m=262144,t=3,p=1$bFF6ZXVHQXY5YTQxdGRMaA$zDZQHBImYcnxNkP5o5iSaA
$ hydra-create-user alice8 --password-hash '$argon2id$v=19$m=262144,t=3,p=1$bFF6ZXVHQXY5YTQxdGRMaA$zDZQHBImYcnxNkP5o5iSaA' --role admin
creating new user `alice8'
```

and then tried logging in with `alice8:hii` and it succeeded!